bin/ghe-backup

```diff
@@ -65,7 +65,7 @@ if [ -f ../in-progress ]; then
   progress=$(cat ../in-progress)
   snapshot=$(echo "$progress" | cut -d ' ' -f 1)
   pid=$(echo "$progress" | cut -d ' ' -f 2)
-  if ! ps -p $pid -o command= | grep ghe-backup; then
+  if ! ps -p $pid -o comm= | grep ghe-backup; then
```

Comment:
@rubiojr seems to be the same on solaris. Never tested this on a running backup. args works on solaris.

```
ghebup@backup:~$ cat data/in-progress 
20170411T140748 7198
ghebup@backup:~$ ps -p 7198 -o comm
COMMAND
bash
ghebup@backup:~$ ps -p 7198 -o args
COMMAND
bash /home/ghebup/backup-utils/bin/ghe-backup -v
```